src/main/resources/application.yml

```diff
@@ -49,4 +46,6 @@ spring:
           aad:
             authorization-uri: ${AAD_ISSUER_URI}/oauth2/v2.0/authorize
             token-uri: ${AAD_ISSUER_URI}/oauth2/v2.0/token
-            jwk-set-uri: ${AAD_ISSUER_URI}/discovery/v2.0/keys
\ No newline at end of file
+            jwk-set-uri: ${AAD_ISSUER_URI}/discovery/v2.0/keys
+server:
+  forward-headers-strategy: native
```

Comment:
Fra wiki:

> The server.use-forward-headers property has been deprecated in favor of server.forward-headers-strategy; previously server.use-forward-headers=true would use the web server native support. You can now achieve the same with server.forward-headers-strategy=native. Because each server has a specific behavior, we now offer an alternative which relies on Spring’s ForwardedHeaderFilter: server.forward-headers-strategy=framework. Developers should use the option that fits best their use case.